models/public/netvlad/netvlad.md

```diff
@@ -0,0 +1,90 @@
+# netvlad
+
+## Use Case and High-Level Description
+
+NetVLAD is a CNN architecture which tackles the problem of large scale visual place recognition. The architecture uses VGG 16 as base network and NetVLAD - a new trainable generalized VLAD (Vector of Locally Aggregated Descriptors) layer. It is a place recognition model pretrained on the [Pittsburgh 250k](http://www.ok.ctrl.titech.ac.jp/~torii/project/repttile/) dataset.
+
+For details see [repository](https://github.com/uzh-rpg/netvlad_tf_open) and [paper](https://arxiv.org/pdf/1511.07247.pdf).
+
+## Specification
+
+| Metric            | Value             |
+|-------------------|-------------------|
+| Type              | Place recognition |
+| GFLOPs            | 36.6374           |
+| MParams           | 149.0021          |
+| Source framework  | TensorFlow\*      |
+
+## Accuracy
+
+Accuracy metrics are obtained on a smaller validation subset of Pittsburgh 250k dataset (Pitts30k) containing 10k database images in each set (train/test/validation).  Images were resized to input size.
+
+| Metric              | Value   |
+| ------------------- | ------- |
+| localization_recall | 82.0321%|
+
+## Input
+
+### Original model
+
+Image, name - `Placeholder`,  shape - `1,200,300,3`, format is `B,H,W,C` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
+
+### Converted model
+
+Image, name - `Placeholder`,  shape - `1,3,200,300`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original model
+
+Floating point embeddings, name - `vgg16_netvlad_pca/l2_normalize_1`,  shape - `1,4096`, output data format  - `B,C`, where:
+
+- `B` - batch size
+- `C` - vector of 4096 floating points values, local image descriptors
+
+### Converted model
+
+The converted model has the same parameters as the original model.
```

Comment:
It is better to duplicate text, especially when it is short

Reply:
Done